src/System.DirectoryServices.AccountManagement/tests/PrincipalContextTests.cs

```diff
@@ -90,7 +93,8 @@ public void Ctor_ContextType_Name_Container_Options(ContextType contextType, str
             }
             else
             {
-                Assert.Throws<InvalidCastException>(() => context.ConnectedServer);
+                Assert.NotNull(context.ConnectedServer);
```

Comment:
Nit: you don't need this as it's handled by the equality check below (and actualy would make the error message a bit worse)